### ISSUE TITLE
Unable to import python files in Lambda python function when using amplify mock

### ISSUE THREAD

**Issue by suneelmatham** [aws-amplify/amplify-cli#6055]
>**Note**: If your issue/bug is regarding the AWS Amplify Console service, please log it in the 
[Amplify Console GitHub Issue Tracker](https://github.com/aws-amplify/amplify-console/issues)

**Describe the bug**
Unable to import python files in Lambda python handler function 'index.py' when using amplify mock. When testing from Lambda console after pushing to the cloud, the function works correctly and python file is getting imported in the Lambda console but still fails in local when using mock .

**Amplify CLI Version**
4,37.0

**To Reproduce**
1. Create a Lambda function in amplify using 'amplify add function' and choose a python backend
2. Create another python file in the same 'src' folder as the handler function index.py. Ex- 'extract.py'
3. Import the python file in handler function 'index.py'
4. Run 'amplify mock function function-name'

**Expected behavior**
Able to import files and run functions in handler function index.py

**Screenshots**
![image](https://user-images.githubusercontent.com/54971275/101079203-2b8a8c00-35cd-11eb-8586-b49c381e7d90.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - Node Version. v15.0.1

**Additional context**

**Comment by half2me** [comment]
@suneelmatham I'm having the exact same issue. I can't import anything, I can't use any libraries.
Also `os.getenv()` does not have the proper variables set like in the nodejs runtime.
Python runtime is basically unusable with `amplify mock`

**Comment by suneelmatham** [comment]
Yeah, I have tried using Lambda layers to import libraries but the memory isn't enough to load deep learning models. I have begun to use EFS to load libraries into lambda function but I am currently facing issues with using libraries which need C-extensions like torch and numpy. 
Though, I have been able to load environment variables with os.environ['var_name']

**Comment by edwardfoyle** [comment]
Hi @suneelmatham @half2me I was able to reproduce this issue. While we work on a fix, you can work around the issue with the following steps:
1. Create a `setup.py` file in `amplify/backend/function/<your function name>/src` with the following contents
```
from distutils.core import setup

setup(name='src', version='1.0')
```
2. Run `pipenv install -e src` from `amplify/backend/function/<your function name>`

This will add an entry to the Pipfile which will make the virtual env (which is used by mock) aware of the files in the src folder

**Comment by vincent38wargnier** [comment]
This solution is causing me timeout errors. Anytime I call my lambda, it's trying to install the dependencies witch takes a while sometimes.

**Comment by edwardfoyle** [comment]
@vincent38wargnier we are aware of that issue and are tracking it as part of [this project](https://github.com/aws-amplify/amplify-cli/projects/17). In the meantime, you can increase the timeout when mocking by using the `--timeout` flag. eg. `amplify mock function myFunction --timeout 30` will wait 30 seconds for a response

**Comment by yuth** [comment]
This is fixed in CLI 4.42.0
